src/index.html

```diff
@@ -16,7 +16,11 @@
       href="./styles/index.scss"
     />
   </head>
-  <body>
-    <h1>Stop watch</h1>
+  <body class="page">
+    <div class="stopwatch stopwatch--speed-up">
+      <div class="stopwatch__minutes-hand"></div>
+      <div class="stopwatch__seconds-hand"></div>
+      <div class="stopwatch__center"></div>
+    </div>
```

Comment:
Ensure there is a corresponding JavaScript functionality that interacts with these classes to implement the stopwatch functionality. If there's no such functionality, you might need to add it as per the project requirements.